src/restapi/fConfig.ts

```diff
@@ -43,3 +54,15 @@ export async function fSubmitOrUpdateZendeskConfigSubmit(req: Request, res: Resp
     }
     res.json(callResponse);
 }
+
+const verifyUrl = async (url: string) => {
+    const verifyURL = url + Routes.ZD.AccessURI;
+    try {
+        const resp = await fetch(verifyURL, {method: 'post'});
```

Comment:
Since the method is `POST`, showing the (clickable) URL in the error message doesn't necessarily help because clicking the URL will have the user send a `GET` request to zendesk, in case they wanted to test it manually. 0/5 on whether that matters

Reply:
done.